### PR TITLE
breaking: Migrate to oriflame package.

### LINKED ISSUE #5: Migration steps
- [x] Update readme; Done by #2 
- [x] Update configs to allow azure devops stuff; Done by #2 
- [x] Setup policies
- [x] Cleanup tags

### LINKED ISSUE #9: Migrate to monorepo
Migrate to monorepo with https://github.com/Oriflame/conventional-changelog-lint-config-ori

### PR DESCRIPTION
**Pull request by rajzik**
- Migration to Oriflame scope
- Migration to monorepo

closes #5 
closes #9

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[**.{js,jsx,ts,tsx,css,json,md}]
+indent_style = spaces
+indent_size = 2
+trim_trailing_whitespace = true
```
Comment (cfmano) [inline .editorconfig:11]: We say that we need to add final newline, but this file is missing it. Maybe would be great to align

Comment (rajzik) [reply .editorconfig:11]: Yeah sure, it is handled by prettier but I will add it here as well.

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    branches:
+      - master
+  pull_request:
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10, 12, 14]
```
Comment (cfmano) [inline .github/workflows/integration.yml:13]: Do we need to support v.10 here? I would go to max. LTS version

File: package.json [hunk 3]
```diff
@@ -62,5 +39,20 @@
...
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^11.0.0",
+    "@rajzik/config-babel": "^3.0.0",
+    "@rajzik/config-eslint": "^4.1.0",
+    "@rajzik/config-jest": "^3.0.0",
+    "@rajzik/config-prettier": "^2.1.4",
+    "@rajzik/config-typescript": "^3.0.0",
+    "@oriflame/conventional-changelog-lint-config": "^0.0.0",
+    "@rajzik/lumos": "^5.0.3",
```
Comment (cfmano) [inline package.json:51]: Are these scoped packages gonna be in the round of switching process?

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Most of the dependencies are updated automatically with dependabot.
+
+Otherwise follow this guide: To update dependencies in conventional-changelog-tools, run
+`yarn upgrade-interactive --latest` in the root of the project. This will display an interactive
+menu in which you can select dependencies to update (to the latest version) using the space bar and
+arrow keys.
+
+However, there are a few caveats and requirements to follow when updating.
+
+1. Dependencies should be updated in isolation per package (workspace), and commited to git
```
Comment (cfmano) [inline .github/CONTRIBUTING.md:66]: a typo: committed

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+   separately. For example, only dependencies within the `@oriflame/conventional-changelog` package should be
+   updated within a single commit.
+   - This allows packages to update and version independently from each other.
```
Comment (cfmano) [inline .github/CONTRIBUTING.md:69]: I don't understand this formulation very well. Maybe that `and` is a left over?

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+   - The exception to this is when a dependency is used across many packages, like Babel being used
+     in Jest and Webpack. The preferred solution is to update all Babel dependencies first, across
+     all packages, in the same commit. Subsequent commits would isolate Jest and Webpack separately.
+2. Format your commit message with the `deps` prefix, along the lines of
+   `deps: Update plugins to latest.` or `deps(ESLint): Update to v6.`. This message format follows
+   the [@oriflame/conventional-changelog][@oriflame/conventional-changelog] spec.
+3. Commit all dependency updates in the same branch, so they can be grouped under the same PR. Once
+   the PR has been approved, normal merge (_not a squash or rebase merge_) the PR so all commits and
```
Comment (cfmano) [inline .github/CONTRIBUTING.md:77]: onsider change of the formulation: 
- merge the PR normally
- merge the PR in standard way
- do non-fast-forward merge

File: .gitignore [hunk 7]
```diff
@@ -33,3 +33,7 @@ tsconfig.json
 webpack.config.js
 tsconfig.eslint.json
 prettier.config.js
+TEST-RESULTS.xml
+*.tsbuildinfo
+tsconfig.json
+tsconfig.options.json
```
Comment (cfmano) [inline .gitignore:39]: please, add new line

File: lerna.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+        "./.github/**",
+        "./.dependabot/**",
+        "./types/**",
+        "*.md",
+        "*.test*"
+      ],
+      "message": "ci: Release [ci skip]"
+    }
+  }
+}
```
Comment (cfmano) [inline lerna.json:17]: is this file autogenerated? if not, then, please, add new line at the end

File: package.json (hunk 3 above)
Comment (rajzik) [reply package.json:51]: This will be in another PR after oriflame lumos is released

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (rajzik) [reply .github/CONTRIBUTING.md:69]: version is used as a verb so it should make sense.

File: packages/conventional-changelog/src/parserOpts.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
+import { ParserOptions } from '@oriflame/conventional-changelog-types';
+import { COMMIT_FORMAT_PREFIX, AZURE_DEVOPS_PREFIX } from './constants';
+
+const options: Partial<ParserOptions> = {
+  headerCorrespondence: ['azurePrefix', 'pr', 'type', 'scope', 'message'],
+  // Keep in sync with checkCommitFormat
+  headerPattern: new RegExp(
+    `${AZURE_DEVOPS_PREFIX.source}${COMMIT_FORMAT_PREFIX.source} (.*)$`,
```
Comment (cfmano) [inline packages/conventional-changelog/src/parserOpts.ts:8]: Shouldn't rather be `(.+)` used?

**Review by cfmano — APPROVED** [review]

File: packages/conventional-changelog/src/parserOpts.ts (hunk 9 above)
Comment (rajzik) [reply packages/conventional-changelog/src/parserOpts.ts:8]: This is from the original repo. I will leave it for now, I might have to remove the end of the line after testing.

**Pull request merged**